customer-commons/src/main/java/no/unit/nva/customer/model/interfaces/DoiAgent.java

```diff
@@ -0,0 +1,23 @@
+package no.unit.nva.customer.model.interfaces;
+
+public interface DoiAgent {
+
+    String getName();
+
+    String getPrefix();
+
+    static DoiAgent randomDoiAgent(String randomString) {
```

Comment:
Test code? Should that be in main package or test package? Ask other reviewer about it

Reply:
Can you move this to a class in the test sources please?